src/axom/sidre/core/View.cpp

```diff
@@ -280,6 +283,71 @@ View* View::reallocate(const DataType& dtype)
   return this;
 }
 
+/*
+ *************************************************************************
+ *
+ * Reallocate data to a different allocator.
+ *
+ *************************************************************************
+ */
+View* View::reallocateTo(int newAllocId)
+{
+  if(newAllocId == axom::INVALID_ALLOCATOR_ID)
+  {
+    return this;
+  }
+
+  if(m_state == EMPTY)
+  {
+    return this;
+  }
+
+  // TODO: Probably wrong to use owning group's default allocator.
+  // Group can allocate View with non-default allocator.
+  // int currentAllocId = getOwningGroup()->getDefaultAllocatorID();
```

Comment:
It looks like this commented-out section has been resolved and the TODO can be removed.

Reply:
Correct.  Thanks for catching.